pages/assets/js/containers.js

```diff
@@ -282,6 +296,58 @@ function drawCpuUsageBreakdown(elementId, machineInfo, containerInfo) {
   drawLineChart(titles, data, elementId, 'Cores');
 }
 
+// Draw the graph for per-subcontainer CPU usage.
+function drawCpuPerSubcontainerUsage(elementId, subcontainerInfos) {
+  if (!hasResourceForAll(subcontainerInfos, 'cpu')) {
+    return;
+  }
+
+  var titles = ['Time'];
+  var data = [];
+  for (var i = 0; i < subcontainerInfos.length; i++) {
+    titles.push(subcontainerInfos[i].name);
+    for (var j = 1; j < subcontainerInfos[i].stats.length; j++) {
+      var cur = subcontainerInfos[i].stats[j];
+      var prev = subcontainerInfos[i].stats[j - 1];
+      var intervalInNs = getInterval(cur.timestamp, prev.timestamp);
+
+      var elements = [cur.timestamp];
+      subcontainerInfos.forEach(function() {
+        elements.push(null);
```

Comment:
For the subcontainers, the timestamps of the metrics are not aligned. If I understand the DataTables API correctly, this means I need to generate a series of sparse arrays of equal length, one for each unique timestamp. Ex:

[timestamp1, null, null, 23],
[timestamp2, null, 322, null],
[timestamp3, 664, null, null],
...